packages/wrangler/src/config/validation-pages.ts

```diff
@@ -38,6 +38,7 @@ const supportedPagesConfigFields = [
 	"browser",
 	// normalizeAndValidateConfig() sets this value
 	"configPath",
+	"upload_source_maps",
```

Comment:
I'm assuming this requires a docs update as well. Pls don't forget to reference it to this PR 🙏

Reply:
Created a docs PR! https://github.com/cloudflare/cloudflare-docs/pull/14929